store/sqlstore/upgrade.go

```diff
@@ -960,12 +960,19 @@ func upgradeDatabaseToVersion531(sqlStore *SqlStore) {
 	}
 }
 
+const RemoteClusterSiteURLUniqueIndex = "remote_clusters_site_url_unique"
```

Comment:
Is there a need for this to be exported?

Reply:
yeap, we are accessing it inside the app, so I wanted to enforce consistency instead of using raw strings. https://github.com/mattermost/mattermost-server/blob/9f2cedae1362308e4ce77dc8c0c795f49575c9ce/app/remote_cluster.go#L20

Reply:
Ah must have missed that. Thanks.